packages/devtools_app/linux/.gitignore

```diff
@@ -1,7 +1 @@
-app_configuration.mk
-Makefile
-flutter
 flutter/ephemeral
```

Comment:
were these changes intentional?

Reply:
Yes is necessary for latest Linux desktop.